cmd/constants.go

```diff
@@ -9,4 +9,7 @@ const (
 	DEFAULT_ACRA_PORT                 = 9393
 	DEFAULT_ACRA_API_PORT             = 9090
 	DEFAULT_ACRA_CONNECTION_PROTOCOL  = "tcp"
+	DEFAULT_ACRA_CONFIGUI_HOST        = "127.0.0.1"
+	DEFAULT_ACRA_CONFIGUI_PORT        = 8000
+	DEFAULT_ACRA_CONFIGUI_STATIC      = "./cmd/acra_configui/static"
```

Comment:
relative path without `./` correctly handled in golang functions that operate with files and paths

Reply:
done